content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+users could write adapters to ingest these attributes and enforce policies or
+transform them before exporting it to a third-party monitoring tool.
+
+While this provided a flexible architecture, mixer consumed significant resources
+in clusters with large amounts of workloads and often became a single point of failure.
+Additionally, the remote call made by the mixer client at Envoy proxies
+attributed to performance degradation and telemetry loss. In Istio 1.3 and
+above, mixer-less aka telemetry v2 support was added to address these concerns
+while surfacing similar telemetry and extension capabilities as telemetry v1.
+Telemetry v2 works by adding WASM extensions in Envoy proxies for surfacing
+Prometheus metrics, thereby removing the out-of-process call to a centralized
+mixer component. Additionally, custom WASM filters can be used to provide mixer
+adapter like functionality. You can read more about this in the announcement
+[blog](/blog/2020/wasm-announce/).
+
+Moving to telemetry v2 reduces resource cost and improves proxy performance, and
+is the recommended way to gather telemetry from Istio, leading us to deprecate
+support for telemetry v1 in Istio 1.7 release. However, there are some
+differences or gaps in the telemetry reported by telemetry v2 as compared to
+telemetry v1 which are important to understand and are discussed below:
+
+1. Telemetry v2 relies on metadata exchange between Envoy proxies to gather
+   information like peer workload name, namespace and labels. In telemetry v1
+   this functionality was performed by mixer as part of combining attributes with
+   the platform data. This metadata exchange in telemetry v2 is performed by the
+   Envoy proxies by adding specific HTTP header for HTTP protocol or augmenting
+   ALPN protocol for TCP protocol as described [here](/docs/tasks/observability/metrics/tcp-metrics/#understanding-tcp-telemetry-collection).
+   This requires Envoy proxies to be injected at both client & server workloads
+   implying that the telemetry reported when one peer is not in the mesh will be
+   missing peer attributes like workload name, namespace and labels.
+   However, if both peers have proxies injected all the labels mentioned [here](/docs/reference/config/metrics/)
+   are available in the generated metrics.
+
+1. Since metadata exchange for TCP protocol relies on
+   [Istio ALPN protocol](/docs/tasks/observability/metrics/tcp-metrics/#understanding-tcp-telemetry-collection).
+   , mutual TLS (mTLS) needs to be enabled for proxies to exchange metadata
+   successfully. This implies that if mTLS is not enabled in your cluster,
+   telemetry for TCP protocol will not include peer information like
+   workload name, namespace and labels.
+
+1. Telemetry v1 supported customizing buckets for histogram type metrics like
+   request duration and TCP byte sizes. In telemetry v2, there's no such
+   mechanism available. Additionally, the buckets available for latency metrics
```

Comment:
Maybe a link to mixer charts in 1.7 and link to what they are now? That may be too technical, but the buckets now make a lot more sense as they are orders of 10, if I recall.